packages/cli/src/daemon-config.ts

```diff
@@ -166,6 +168,26 @@ export class DaemonAnchorConfig {
   ethereumRpcUrl?: string
 }
 
+@jsonObject
+@toJson
+export class IndexingConfig {
+  @jsonMember(String)
+  db?: string
+
+  @jsonArrayMember(String, {
+    emitDefaultValue: true,
+    deserializer: (arr?: Array<string>) => {
+      if (!arr) return arr
+      return arr.map(StreamID.fromString)
+    },
+    serializer: (arr?: Array<StreamID>) => {
+      if (!arr) return arr
+      return arr.map((s) => s.toString())
+    },
+  })
+  models: string[]
```

Comment:
this seems backwards to me.  Shouldn't the `models` field be of type `StreamID[]`, and the serializer translate a string array into a streamid array?

That would match the way the serializer for `port` works above - the serializer turns the input from the user as a string into the type we want to use internally

Reply:
Other way around. Serialisers were always present in the correct form. Types were a bit off.

Reply:
I guess, `port` one is broken. According to https://github.com/JohnWeisz/TypedJSON#serializer-and-deserializer `deserializer` turns input from JSON text to a runtime object, and `serializer` turns it back to JSON text.

Reply:
Huh.  I wonder then why we haven't seen issues with the `port` serializer.  That was a pretty new change - maybe its broken but no one's noticed yet?  @willex can you double-check this?

Reply:
Yes, looking at it now.